var/spack/repos/builtin.mock/packages/url-list-test/package.py

```diff
@@ -12,9 +14,10 @@ class UrlListTest(Package):
 
     homepage = "http://www.url-list-example.com"
 
+    drive_escape = "/" if sys.platform == "win32" else ""
     web_data_path = join_path(spack.paths.test_path, "data", "web")
-    url = "file://" + web_data_path + "/foo-0.0.0.tar.gz"
-    list_url = "file://" + web_data_path + "/index.html"
+    url = "file://" + drive_escape + web_data_path + "/foo-0.0.0.tar.gz"
+    list_url = "file://" + drive_escape + web_data_path + "/index.html"
```

Comment:
Would `spack.util.url.path_to_file_url` also generate this properly? If so, IMO it would preferable to use that

Reply:
path_to_file_url does generate it properly and therefore I updated the code to include it.